src/tests/dictionaryConcepts/container/EditConcept.test.jsx

```diff
@@ -61,9 +62,42 @@ describe('Test suite for dictionary concepts components', () => {
     fetchAllConceptSources: jest.fn(),
     allSources: [mockSource],
     addSelectedAnswers: jest.fn(),
+    selectConfirm: jest.fn(),
     selectedAnswers: [],
     createNewAnswerRow: jest.fn(),
   };
+
+  it('should call selectConfirm', () => {
```

Comment:
When should select confirm be called?

Reply:
"select confirm" is called when confirming the cancellation on the pop-up cancel modal.

Reply:
Would the maintainer of this code always be looking out for you to get that answer?

Reply:
No, I have made the title of the test more appropriate. Thank you for your observation.